packages/web/components/TechnologyForm/Review/index.js

```diff
@@ -229,6 +230,10 @@ const Review = ({ data: { technology } }) => {
 						/>
 					</Section>
 
+					<Section title="Mapas" color="lightGray" hideWhenIsEmpty={false}>
+						<GeoLocation rawTerms={technology?.rawTerms} isStacked />
```

Comment:
```suggestion
						<GeoLocation rawTerms={technology?.rawTerms} stacked />
```

Acho que fica melhor chamar apenas de `stacked` pois o `isStacked` da a entender como uma pergunta ao inves de afirmação, o que você quer aqui é dizer que o geolocation é stacked. Acho que fica melhor apenas `stacked` 👍   (irá precisar renomear no component obv. rs)

Reply:
@marcusmota boa! Obrigada pelas sugestões!